src/main/java/chess/service/ChessService.java

```diff
@@ -26,112 +38,109 @@ public ChessService(BoardDao boardDao, RoomDao roomDao) {
         this.roomDao = roomDao;
     }
 
-    public BoardDto getInitialBoard(long roomId) {
-        GameState gameState = getGameState(roomId);
-        Map<String, String> board = getBoardPieces(roomId);
-        List<PieceDto> pieces = board.entrySet()
-                .stream()
-                .map(i -> new PieceDto(i.getKey(), i.getValue()))
-                .collect(Collectors.toUnmodifiableList());
-        return new BoardDto(pieces, gameState.getTeam());
+    public void createRoom(CreateRoomDto room) {
+        Long roomId = roomDao.save(room.getTitle(), room.getPassword());
+        boardDao.saveAll(BoardInitialize.create(), roomId);
     }
 
-    private GameState getGameState(long roomId) {
-        RoomDto room = roomDao.findById(roomId);
-        if (room == null) {
-            return createGameState(roomId);
+    public List<RoomDto> findRoomList() {
```

Comment:
```suggestion
    public List<RoomDto> findRooms() {
```

리턴 타입에 list가 있으니 메서드명에 또 써줄 필요는 없겠네요

Reply:
메서드명에 리턴 타입을 쓰는 실수가 좀 보이네요..! 수정했습니다!😂